src/sdk-reference/java/1/getting-started/index.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: sdk.html.hbs
+title: Getting started
+description: Java Getting started
+order: 100
+separator: essentials
+---
+
+# Getting Started
+
+In this tutorial you will learn how to install the Kuzzle **Java SDK**.
+This page shows examples of scripts that **store** documents in Kuzzle, and of scripts that subcribe to real-time **notifications** for each new document created.
+
+<div class="alert alert-success">
+Before proceeding, please make sure your system meets the following requirements:
+
+* **Oracle JDK or OpenJDK** version 8 or higher (<a href="https://openjdk.java.net/install/">OpenJDK installation instructions</a>)
+* A running Kuzzle server (<a href="guide/1/essentials/installing-kuzzle/">Kuzzle installation guide</a>)
+</div>
+
+## Installation
+
+### Maven and Gradle
+
+Kuzzle Java SDK is available on Bintray for both
+[amd64](https://bintray.com/kuzzle/maven/kuzzle-sdk-java-amd64) and
+[x86](https://bintray.com/kuzzle/maven/kuzzle-sdk-java-x86) architectures. Add it as a dependency for your project and
+
+### Manually
+
+If you do not use a project manager, you can find the SDK JARs directly
+on our [download
+plateform](https://dl.kuzzle.io/sdk/java/master/index.html). Download
+and add it to your classpath.
+
+## First connection
+
+Initialize a new Java project, create a `gettingstartedfirstconnection.java` file and start by adding the code below:
+
+[snippet=firstconnection]
+
+This program initializes the Kuzzle server storage by creating a data index, and a data collection inside it
```

Comment:
If we refer to https://confluence.kaliop.net/display/LABS/2018-02-21+Meeting+notes++-+Kuzzle+products+naming, Kuzzle server should be referenced as Kuzzle Server